### DISCUSSION TITLE
circular import problem

### DISCUSSION THREAD

**Discussion by Vanbk** [mrgnr/camus#49] [General]
Hi everyone!

I am trying to rebuild and run the project cloned from the master branch to study. However, I face the following error:
"ImportError: cannot import name 'db' from partially initialized module 'camus' (most likely due to a circular import) (/opt/camus/camus/__init__.py)". Since it is a design error, could you please tell me how can resolve that.

Thank you very much!

**Comment by mrgnr** [comment]
Hi @Vanbk, how are you trying to run Camus?

The following should get a server running on Debian/Ubuntu (with the `python3-virtualenv` and `python3-pip` packages already installed): 

```bash
$ git clone git@github.com:camuschat/camus.git && cd camus
$ virtualenv venv
$ . ./venv/bin/activate
$ pip install -r requirements.txt
$ ./bin/camus
```

The [development documentation](https://docs.camus.chat/en/latest/development.html) might also be helpful for setting up your environment.
